websites/G/Game-Linter/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "/home/achbarou",
+    "id": "563804458526441639"
+  },
+  "service": "Game-Linter",
+  "description": {
+    "en": "Game-Linter is a file sharing and a torrent index as well as a magnet link provider."
+  },
+  "url": "game-linter.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/tQrygFc.png",
+  "thumbnail": "https://i.imgur.com/ISdHrQA.png",
+  "color": "#30ACCE",
+  "tags": ["gaming", "filesharing", "coding", "sharing"],
+  "category": "other",
+  "regExp": "(.*).?game-linter.com(.*)"
```

Comment:
you need regex for what?

Reply:
yes , the website had many subdomains

Reply:
I don't see any on website.

Reply:
should i remove the regex?
![image](https://user-images.githubusercontent.com/39389636/104915009-4d2eaf80-5990-11eb-99b7-e7963a53aba9.png)
these are the subdomains tho

Reply:
i don't think ppl are on it since you only support torrent site, and on this it doesn't have any redirects to these subdomains

Reply:
well then i'll remove the regex

Reply:
Done